### PR TITLE
Migrate codeaurora repositories to github

### PR DESCRIPTION
**Pull request by logbaseaofn**
codeaurora.org has now been archived. This fixes all affected source uris.

### CODE REVIEW

File: recipes-bsp/boot-format/boot-format_git.bb [hunk 1]
```diff
@@ -3,7 +3,7 @@ LICENSE = "GPLv2"
 PR = "r6"
 LIC_FILES_CHKSUM = "file://COPYING;md5=b234ee4d69f5fce4486a80fdaf4a4263"
 
-SRC_URI = "git://source.codeaurora.org/external/qoriq/qoriq-yocto-sdk/boot-format;nobranch=1 \
+SRC_URI = "git://github.com/qoriq-yocto-sdk/boot-format;nobranch=1 \
```
Comment (otavio) [inline recipes-bsp/boot-format/boot-format_git.bb:6]: missing protocol.

File: recipes-extended/ipc-ust/ipc-ust_git.bb [hunk 2]
```diff
@@ -3,7 +3,7 @@ DESCRIPTION = "DSP boot application and ipc test application"
 LICENSE = "BSD"
 LIC_FILES_CHKSUM = "file://COPYING;md5=fa38cd73d71527dc6efb546474f64d10"
 
-SRC_URI = "git://source.codeaurora.org/external/qoriq/qoriq-yocto-sdk/ipc;branch=nxp/sdk-v2.0.x \
+SRC_URI = "git://github.com/qoriq-yocto-sdk/ipc;branch=nxp/sdk-v2.0.x \
```
Comment (otavio) [inline recipes-extended/ipc-ust/ipc-ust_git.bb:6]: missing protocol

File: recipes-extended/tsntool/tsntool_git.bb [hunk 3]
```diff
@@ -7,7 +7,7 @@ DEPENDS = "cjson libnl readline"
 
 inherit pkgconfig
 
-SRC_URI = "git://source.codeaurora.org/external/qoriq/qoriq-components/tsntool;protocol=https;nobranch=1"
+SRC_URI = "git://github.com/nxp-qoriq/tsntool;protocol=https;protocol=https;nobranch=1"
```
Comment (otavio) [inline recipes-extended/tsntool/tsntool_git.bb:10]: duplicated protocol

File: recipes-kernel/kernel-modules/kernel-module-ls-debug_git.bb [hunk 4]
```diff
@@ -7,7 +7,7 @@ LIC_FILES_CHKSUM = "file://COPYING;md5=94263f12f9416f9fd0493c8f9e8085a3"
 
 inherit module autotools-brokensep
 
-SRC_URI = "git://source.codeaurora.org/external/qoriq/qoriq-yocto-sdk/ls-dbg;branch=nxp/master"
+SRC_URI = "git://github.com/qoriq-yocto-sdk/ls-dbg;branch=nxp/master"
```
Comment (otavio) [inline recipes-kernel/kernel-modules/kernel-module-ls-debug_git.bb:10]: missing protocol

File: recipes-kernel/kernel-modules/kernel-module-perf-qoriq_0.8.2.bb [hunk 5]
```diff
@@ -2,7 +2,7 @@ DESCRIPTION = "QorIQ extension to Perf for supporting non core counters"
 LICENSE = "GPLv2+"
 LIC_FILES_CHKSUM = "file://COPYING;md5=e29234dd5d40dc352cc60cc0c93437ba"
 
-SRC_URI = "git://source.codeaurora.org/external/qoriq/qoriq-yocto-sdk/qoriq-perf;branch=nxp/master"
+SRC_URI = "git://github.com/qoriq-yocto-sdk/qoriq-perf;branch=nxp/master"
```
Comment (otavio) [inline recipes-kernel/kernel-modules/kernel-module-perf-qoriq_0.8.2.bb:5]: missing protocol

File: recipes-kernel/kernel-modules/kernel-module-scatter-gather_git.bb [hunk 6]
```diff
@@ -4,7 +4,7 @@ LIC_FILES_CHKSUM = "file://COPYING;md5=e9605a22ea50467bd2bfe4cdd66e69ae"
 
 inherit module
 
-SRC_URI = "git://source.codeaurora.org/external/qoriq/qoriq-yocto-sdk/scatter-gather;branch=nxp/master"
+SRC_URI = "git://github.com/qoriq-yocto-sdk/scatter-gather;branch=nxp/master"
```
Comment (otavio) [inline recipes-kernel/kernel-modules/kernel-module-scatter-gather_git.bb:7]: missing protocol

File: recipes-kernel/kernel-modules/kernel-module-uio-seville_0.1.bb [hunk 7]
```diff
@@ -2,7 +2,7 @@ DESCRIPTION = "UIO driver for T1040 L2 Switch"
 LICENSE = "GPLv2"
 LIC_FILES_CHKSUM = "file://COPYING;md5=12f884d2ae1ff87c09e5b7ccc2c4ca7e"
 
-SRC_URI = "git://source.codeaurora.org/external/qoriq/qoriq-yocto-sdk/l2switch-uio;branch=nxp/sdk-v2.0.x"
+SRC_URI = "git://github.com/qoriq-yocto-sdk/l2switch-uio;branch=nxp/sdk-v2.0.x"
```
Comment (otavio) [inline recipes-kernel/kernel-modules/kernel-module-uio-seville_0.1.bb:5]: missing protocol

**Review by otavio — CHANGES_REQUESTED** [review]

**Comment by logbaseaofn** [comment]
@otavio Right, I will fix the protocols and update the PR, thank you.

**Review by otavio — APPROVED** [review]

**Pull request merged**